app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,8 @@
 class PostsController < ApplicationController
   before_action :index
   def index
-    # @posts = Post.where(author_id: params[:user_id])
-    # # @user = User.find(id:params[:user_id])ß
+    @posts = Post.where(author_id: params[:user_id])
     @user = User.find(params[:user_id])
-    @posts = @user.posts.includes(:comments, comments: :author).order(created_at: :desc)
   end
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍